nltools/data/brain_data.py

```diff
@@ -489,7 +482,8 @@ def scale(self, scale_val=100.):
 
         return out
 
-    def plot(self, limit=5, anatomical=None, view='axial', threshold_upper=None, threshold_lower=None, **kwargs):
+    def plot(self, limit=5, anatomical=None, view='axial', colorbar=False, black_bg=True, draw_cross=False,
```

Comment:
Do you want to add a parameter so `figsize` is controllable with a default of `(15,2)` so we can adjust figure sizing for other use cases (e.g. combining multiple figs, fitting to pages, etc)?

Reply:
oh that's a good idea. also cmap

Reply:
done